rpc/js/tsconfig.json

```diff
@@ -4,11 +4,15 @@
     "target": "ES5",
     "useDefineForClassFields": true,
     "importHelpers": true,
-    "module": "commonjs",
-    "moduleResolution": "Node",
+    "module": "ESNext",
+    "moduleResolution": "node",
     "verbatimModuleSyntax": false,
+    "declaration": true,
+    "emitDeclarationOnly": true,
```

Comment:
[Nit] These options are only for the build, so should be moved to `tsconfig.build.json`